aiida_wannier90/parsers.py

```diff
@@ -208,14 +208,27 @@ def raw_wout_parser(wann_out_file):
                 wf_out_i['wannier_function'] = int(
                     line.split('(')[0].split()[-1]
                 )
-                wf_out_i['spread'] = float(line.split('(')[1].split()[-1])
+                wf_out_i['spread'] = float(line.split(')')[1].strip())
                 #wf_out_i['spread'] = float(line.split()[-1])
-                #x = float(line.split()[-5].strip(','))
-                #y = float(line.split()[-4].strip(','))
-                #z = float(line.split()[-3].strip(','))
-                x = float(line.split('(')[1].split()[0].strip(','))
-                y = float(line.split('(')[1].split()[1].strip(','))
-                z = float(line.split('(')[1].split()[2].strip(','))
+                try:
+                    x = float(
+                        line.split('(')[1].split(')')[0].split(',')[0].strip()
+                    )
+                except (ValueError, IndexError):
+                    # To avoid that the crasher completely fails, we set None as a fallback
+                    x = None
+                try:
+                    y = float(
+                        line.split('(')[1].split(')')[0].split(',')[1].strip()
+                    )
+                except (ValueError, IndexError):
+                    y = None
+                try:
+                    z = float(
+                        line.split('(')[1].split(')')[0].split(',')[2].strip()
+                    )
```

Comment:
I think we could de-duplicate the ``line.split('(')[1].split(')')[0].split(',')`` part:

Maybe something like the following?
```
x, y, z = (
    float(val) for val in 
    line.split('(')[1].split(')')[0].split(',')
)
```
Or is there a case where one of the coordinates could fail, but not the others?

Reply:
There is unfortunately, when ******* is printed. We will need anyway at some point to change Wannier90 to print what we need in a csv file or similar as discussed in #1